test-exceptions/try-text.wast

```diff
@@ -0,0 +1,13 @@
+;; out/clang/Debug/wast2wasm --except test-exceptions/try-text.wast
```

Comment:
Actually, it just represents the manual command to run, since I don't know how to use the scripting language.